add_array_built_in.rb

```diff
@@ -0,0 +1,8 @@
+arr1 = [1,2,3,5]
+arr2 = [6,7,8,9]
+arr3 = arr1.concat arr2
```

Comment:
@mohseengs why did we used concat and not any other method? Do you know how exactly concat works.

Try printing arr1 and arr2 also then you will get the idea.

Reply:
concat method we can use to join to arrays,  it actually append elements of array that is passed as an argument to the array on which the concat method is called.
we can also use + operator to join arrays the difference will be it will not change any array, rather it returns a new one